examples/AspNetCore/Program.cs

```diff
@@ -30,7 +31,13 @@ public static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
-                    webBuilder.UseStartup<Startup>();
+                    var config = new ConfigurationBuilder()
```

Comment:
lets just stick the default config, and put this setting inside appsettings.json. I believe the current change breaks settings which are retrieved from appsettings.json now.

Reply:
Yep! You're completely right, apologies, I was making a hasty change.